cypress/support/execute-tests.sh

```diff
@@ -159,6 +167,13 @@ function overrideSettings() {
   sed -i.bak 's@STREET_MAP_URL@'$STREET_MAP_URL'@g' $OPENSPHERE_CONFIG_TESTER && rm $OPENSPHERE_CONFIG_TESTER.bak
   sed -i.bak 's@WORLD_IMAGERY_URL@'$WORLD_IMAGERY_URL'@g' $OPENSPHERE_CONFIG_TESTER && rm $OPENSPHERE_CONFIG_TESTER.bak
 
+  echo 'INFO: zoom offset to settings file'
+  if [ "$CYPRESS_PROJECTION" = 3857 ]; then
+    sed -i.bak 's@"ZOOM_OFFSET"@'0'@g' $OPENSPHERE_CONFIG_TESTER && rm $OPENSPHERE_CONFIG_TESTER.bak
```

Comment:
Why add the `-i.bak` option if you are just going to subsequently delete the `.bak` file in the next statement?

Reply:
I don't believe there was a good reason for that. Removed.

Reply:
The `-i.bak` flag was for cross-platform support, so we should bring that back for now. Details here:
https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux

Reply:
Kevin thought it was related to https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux

I have the GNU coreutils installed on mine so I don't see the difference, but out of the box, removing that option may not work.

Reply:
Commit reverted